addons/l10n_es/data/template/account.fiscal.position-es_common.csv

```diff
@@ -1,816 +1,816 @@
-"","","","","","","","","","account_tax_template_s_iva21isp","account_tax_template_s_iva21isp"
-"","","","","","","","","","account_tax_template_s_iva21isp","account_tax_template_s_irpfnrnue0"
-"","","","","","","","","","account_tax_template_s_iva21s","account_tax_template_s_iva21s"
-"","","","","","","","","","account_tax_template_s_iva21s","account_tax_template_s_irpfnrnue0"
-"","","","","","","","","","account_tax_template_s_iva10b","account_tax_template_s_iva10b"
-"","","","","","","","","","account_tax_template_s_iva10b","account_tax_template_s_irpfnrnue0"
-"","","","","","","","","","account_tax_template_s_iva10s","account_tax_template_s_iva10s"
-"","","","","","","","","","account_tax_template_s_iva10s","account_tax_template_s_irpfnrnue0"
-"","","","","","","","","","account_tax_template_s_iva4b","account_tax_template_s_iva4b"
-"","","","","","","","","","account_tax_template_s_iva4b","account_tax_template_s_irpfnrnue0"
-"","","","","","","","","","account_tax_template_s_iva4s","account_tax_template_s_iva4s"
-"","","","","","","","","","account_tax_template_s_iva4s","account_tax_template_s_irpfnrnue0"
-"","","","","","","","","","account_tax_template_p_iva21_bc","account_tax_template_p_iva21_bc"
-"","","","","","","","","","account_tax_template_p_iva21_bc","account_tax_template_p_irpfnrnue0p"
-"","","","","","","","","","account_tax_template_p_iva21_sc","account_tax_template_p_iva21_sc"
-"","","","","","","","","","account_tax_template_p_iva21_sc","account_tax_template_p_irpfnrnue0p"
-"","","","","","","","","","account_tax_template_p_iva10_bc","account_tax_template_p_iva10_bc"
-"","","","","","","","","","account_tax_template_p_iva10_bc","account_tax_template_p_irpfnrnue0p"
-"","","","","","","","","","account_tax_template_p_iva10_sc","account_tax_template_p_iva10_sc"
-"","","","","","","","","","account_tax_template_p_iva10_sc","account_tax_template_p_irpfnrnue0p"
-"","","","","","","","","","account_tax_template_p_iva4_bc","account_tax_template_p_iva4_bc"
-"","","","","","","","","","account_tax_template_p_iva4_bc","account_tax_template_p_irpfnrnue0p"
-"","","","","","","","","","account_tax_template_p_iva4_sc","account_tax_template_p_iva4_sc"
-"","","","","","","","","","account_tax_template_p_iva4_sc","account_tax_template_p_irpfnrnue0p"
-"","","","","","","","","","account_tax_template_s_iva0","account_tax_template_s_iva0"
-"","","","","","","","","","account_tax_template_s_iva0","account_tax_template_s_irpfnrnue0"
-"","","","","","","","","","account_tax_template_p_iva0_bc","account_tax_template_p_iva0_bc"
-"","","","","","","","","","account_tax_template_p_iva0_bc","account_tax_template_p_irpfnrnue0p"
\ No newline at end of file
+"id","sequence","name","name@es","name@ca","auto_apply","vat_required","country_id","country_group_id","tax_ids/tax_src_id","tax_ids/tax_dest_id","account_ids/account_src_id","account_ids/account_dest_id","state_ids"
```

Comment:
If I'm not mistake, we don't do anything with `state_ids` here. Why are we adding it ?